app/controllers/roles_controller.rb

```diff
@@ -46,8 +46,33 @@ def direct_memberships
     render json: json
   end
   
+  # Find all members of this role.
+  #
+  # For each member, return the full details of the grant as a
+  # JSON object
+  #
+  # +params[:count] returns only the number of members
+  # +params[:limit] and [:offset] control the paging
+  # +params[:search] returns only the members that match the search string
+  def members
+    filter_options, render_options = members_options
```

Comment:
I'd just make each of this its own method, rather than using array destructuring + the array returning `members_options` method